pio/ws2812/ws2812_parallel.c

```diff
@@ -300,7 +302,7 @@ int main() {
         int brightness = 0;
         uint current = 0;
         for (int i = 0; i < 1000; ++i) {
-            int n = 64;
+            int n = NUM_PIXELS;
 
             current_string_out = string0.data;
             current_string_4color = false;
```

Comment:
Might be worth making this `current_string_4color = IS_WRGB;` (and adding the define) ? :shrug: 

And I'm sure no-one would mind if you fixed the spelling to `current_string_4colour` :laughing:

Reply:
It seems it is possible to control a WRGB and RGB strip at the same time, so a single define will not do it. Regarding color vs colour: I can do this.

Reply:
You're right, I hadn't looked at the code properly :wink: